base/applications/mspaint/winproc.cpp

```diff
@@ -841,12 +840,22 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
         }
         case IDM_EDITCOPYTO:
         {
-            WCHAR szFileName[MAX_LONG_PATH] = L"*.png";
+            WCHAR szFileName[MAX_LONG_PATH] = L"";
+            CStringW strUntitled(MAKEINTRESOURCEW(IDS_DEFAULTFILENAME));
+            lstrcpynW(szFileName, strUntitled, _countof(szFileName));
```

Comment:
safe-string function instead?

Reply:
Used `#include <strsafe.h>` and `StringCchCopyW`.

Reply:
Used `#include <strsafe.h>` and `StringCchCopyW`.